cli/Cargo.toml

```diff
@@ -21,14 +21,14 @@ chrono = { version = "0.4.35", default-features = false, features = ["clock"] }
 clap = { version = "4.4", features = ["derive", "env"] }
 comfy-table = "7.1"
 csv = "1.3"
+databend-common-ast = "0.0.0"
```

Comment:
The package should have a valid version number.

Reply:
it's a valid lastest version